src/Chapter1.hs

```diff
@@ -636,7 +664,14 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | (abs n) < 10 = (abs n)
+  | otherwise = 
+    let absN = abs n
+        l = absN `mod` 10
+        newNum = (absN - l) `div` 10
```

Comment:
Very cool 😎 

When you do `div`, no need to subtract the `mod` 🙂 

Also, if you put `absN` into the `where` clause, instead of the `let` in the second guard, you could reuse it in the first case as well 👍🏼